lib/Crypto/Util/asn1.py

```diff
@@ -475,7 +475,7 @@ def encode(self):
                         self.payload += item.encode()
                 return DerObject.encode(self)
 
-        def decode(self, der_encoded, nr_elements=None, only_ints_expected=False, strict=False):
```

Comment:
Hmm, changed signature, may result in compatibility issues down the road? No?

Reply:
I thought about that afterwards, too. Not my decision to make, I changed it to be in line with the type stubs I created (and the class hierarchy). Depending on the priorities, it could be changed back.